nntrainer/src/conv2d_layer.cpp

```diff
@@ -164,19 +154,20 @@ Tensor Conv2DLayer::backwarding(Tensor derivative, int iteration) {
           derivative.getAddress(b * derivative.getDim().getFeatureLen() +
                                 i * derivative.height() * derivative.width()),
           in_dim, output.data(), stride, 0.0);
-        memcpy(delK.getAddress(b * delK.getDim().getFeatureLen() + j * o_size),
-               output.data(), o_size * sizeof(float));
+        float *del = delK.getAddress(j * o_size);
+        for (unsigned k = 0; k < o_size; ++k) {
+          del[i] += output[i];
```

Comment:
Can `/// @fixme: #280` go away?

Reply:
yes. I think so. we are not going to use gradient with batch size from now on.